src/question_tree.rb

```diff
@@ -45,20 +46,38 @@ def identifiers
       filtered_each.map(&:identifier)
     end
 
+    # START REMAPPING 'content' to variable names
```

Comment:
I don't think we need these markers indicating what this group of different methods is for, it already seems pretty clear to me what these methods do and adding this is just extra noise which needs to be read. If you really want to keep these methods distinct from others you could extract a well-named mixin for them, for which [`ActiveSupport::Concern`](http://devdocs.io/rails~5.1/activesupport/concern) could be used if you wanted, but I don't think that's necessary.

Reply:
Done